django_rq_wrapper/management/commands/rqworkers.py

```diff
@@ -76,8 +61,7 @@ def handle(self, *args, **options):
 
         pid = options.get('pid')
         if pid:
-            with open(os.path.expanduser(pid), "w") as fp:
-                fp.write(str(os.getpid()))
+            Path(pid).expanduser().write_text(str(os.getpid()))
```

Comment:
This isn't backwards compatible with pre Python 3.4, but that is probably ok at this point